data/fcitx5-diagnose.sh.in

```diff
@@ -741,16 +728,8 @@ init_ld_paths() {
 }
 init_ld_paths
 
-fcitx_lib_path=()
-init_fcitx_lib_path() {
-    local path
-    local __fcitx_lib_path
-    fcitx_lib_path=(/usr/lib/fcitx5/ /usr/local/lib/fcitx5/)
```

Comment:
Can we just extend this line with a FCITX_INSTALL_ADDONDIR entry and keep the existing implementation?